subsys/bluetooth/controller/ll_sw/lll_filter.h

```diff
@@ -7,11 +7,21 @@
 #define WL_SIZE            8
 #define FILTER_IDX_NONE    0xFF
 
-struct ll_filter {
+struct lll_filter {
 	u8_t  enable_bitmask;
 	u8_t  addr_type_bitmask;
 	u8_t  bdaddr[WL_SIZE][BDADDR_SIZE];
 };
 
-struct ll_filter *ctrl_filter_get(bool whitelist);
-void ll_adv_scan_state_cb(u8_t bm);
+extern bt_addr_t *ull_filter_lll_lrpa_get(u8_t rl_idx);
```

Comment:
why can't these functions be just named `lll_*()` without the `ull_filter_` prefix?

Reply:
I think the notation is that this is an interface provided by ull_filter but is executed in lll context